documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -48,34 +52,32 @@ metadata:
   #...
 type: Opaque
 ----
-<1> Current CA certificate base64 encoded
-<2> Current CA certificate generation
+<1> Current base64-encoded CA certificate
+<2> Current CA certificate generation annotation value
 
-.. Encode your new CA certificate into base64.
+. Encode your new CA certificate into base64.
 +
 [source,shell,subs="+quotes"]
-cat _PATH-TO-NEW-CERTIFICATE_ | base64
+cat _<path_to_new_certificate>_ | base64
 
-.. Update the CA certificate.
+. Update the CA certificate.
 +
-Copy the new CA certificate base64 encoded got from the previous step into the `ca.crt` field of the `data` section in the `Secret`.
+Copy the base64-encoded CA certificate from the previous step as the value for the `ca.crt` property under `data`.
 +
-.. Increase the CA certificate generation.
+. Increase the value of the CA certificate generation annotation.
 +
 Update the `strimzi.io/ca-cert-generation` annotation with a higher incremental value.
 For example, change `strimzi.io/ca-cert-generation=0` to `strimzi.io/ca-cert-generation=1`.
-If the `Secret` is missing the annotation, it is treated as generation `0` so add the annotation with `1` as value.
-
+If the `Secret` is missing the annotation, the value is treated as `0`, so change the value to `1`.
```

Comment:
why "change"? if it's missing you have to add the annotation, you cannot change it.